TeachingRecordSystem/src/TeachingRecordSystem.Api/V3/Core/Operations/GetTrnRequest.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace TeachingRecordSystem.Api.V3.Core.Operations;
 
-public record GetTrnRequestCommand(string RequestId);
+public record GetTrnRequestCommand(string RequestId, bool? ResolveMerges = true);
```

Comment:
Doesn't look like we're using `ResolveMerges`...
```suggestion
public record GetTrnRequestCommand(string RequestId);
```